cosmosdb/cassandra/ps-cassandra-ru-get.ps1

```diff
@@ -0,0 +1,32 @@
+# Get RU for an Azure Cosmos Cassandra API keyspace or table
+$apiVersion = "2015-04-08"
+$resourceGroupName = "myResourceGroup"
+$accountName = "mycosmosaccount"
+$keyspaceName = "keyspace1"
+$tableName = "table1"
+$keyspaceThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/keyspaces/settings"
+$tableThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/keyspaces/tables/settings"
+$keyspaceThroughputResourceName = $accountName + "/cassandra/" + $keyspaceName + "/throughput"
+$tableThroughputResourceName = $accountName + "/cassandra/" + $keyspaceName + "/" + $tableName + "/throughput"
+
+# Get the throughput for a keyspace (returns RU/s or error if not set)
+Get-AzResource -ResourceType $keyspaceThroughputResourceType `
+    -ApiVersion $apiVersion -ResourceGroupName $resourceGroupName `
+    -Name $keyspaceThroughputResourceName | Select-Object Properties
+
+if($error[0].Exception.Message.Split(",")[0].Split(":")[1].Replace("`"","") -eq "NotFound")
+{
+    Write-Host "Throughput not set on keyspace resource"
+    $error.Clear()
+}
+
+# Get the throughput for a table (returns RU/s or error if not set)
+Get-AzResource -ResourceType $tableThroughputResourceType `
+    -ApiVersion $apiVersion -ResourceGroupName $resourceGroupName `
+    -Name $tableThroughputResourceName | Select-Object Properties
+
+if($error[0].Exception.Message.Split(",")[0].Split(":")[1].Replace("`"","") -eq "NotFound")
```

Comment:
If there is _not_ an error, this displays an error:

```
You cannot call a method on a null-valued expression.
At C:\Users\sttramer\src\azure-docs-powershell-samples\cosmosdb\cassandra\ps-cassandra-ru-get.ps1:28 char:4
+ if($error[0].Exception.Message.Split(",")[0].Split(":")[1].Replace("` ...
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull
```

If you're going to perform an explicit error check rather than include information for the user to determine if their response is due to a missing setting vs. genuine error, please make sure that the error check itself does not throw an error.

This comment applies to all of the added error checks.